stories/components/social-nav/social-nav.content.json

```diff
@@ -0,0 +1,23 @@
+{
```

Comment:
@mejiaj The fallback path does not seem to be working for this and the icon_paths are not set here, I'm seeing:
![image](https://github.com/Bixal/bixal-site-drupal/assets/121190622/41575aea-0124-4ec3-ba36-a8631ebd6229)

Reply:
@mpbixal 

Probably tried being too clever. The intention here was to _try_ to set a good default in StorybookJS. I see now it's not helpful anywhere else. Removed in cddba22.

Open to suggestions on improving this experience in Drupal. I've removed this and explicitly defined the path in `social-nav.content.json`.

---

_I wasn't able to reproduce with `npm run storybook:local` or `http://storybook.bixalcom.lndo.site/`._